es_templates/sku_insta_search_v0

```diff
@@ -0,0 +1,208 @@
+                        "boost": 10,
+                        "query": {
+                          "match": {
+                            "attributes.pack_type.exhaustive": "{{query}}"
+                          }
+                        }
+                      }
+                    },
+                    {
+                      "nested": {
+                        "path": "attributes",
+                        "boost": 9,
+                        "query": {
+                          "match": {
+                            "attributes.form_factor.exhaustive": "{{query}}"
+                          }
+                        }
+                      }
+                    },
+                    {
+                      "nested": {
+                        "path": "category",
+                        "query": {
+                          "multi_match": {
+                            "query": "{{query}}",
+                            "fields": ["category.name.exhaustive"],
+                            "boost": 8
+                          }
+                        }
+                      }
+                    },
+                    {
+                      "nested": {
+                        "path": "attributes",
+                        "query": {
+                          "multi_match": {
+                            "query": "{{query}}",
+                            "fields": [
+                              "attributes.type.exhaustive",
+                              "attributes.product_name.exhaustive",
+                              "attributes.brand.exhaustive",
+                              "attributes.model_name.exhaustive",
+                              "attributes.sub_category/L3.exhaustive",
+                              "attributes.category/I2.exhaustive",
+                              "attributes.category/L2.exhaustive",
+                              "attributes.super_category/I1.exhaustive",
+                              "attributes.sub_category/I1.exhaustive",
+                              "attributes.pack_type.exhaustive",
+                              "attributes.form_factor.exhaustive"
+                            ]
+                          }
+                        }
+                      }
+                    },
+                    {
+                      "nested": {
+                        "path": "attributes",
+                        "boost" : 2,
+                        "query": {
+                          "prefix": {
+                            "attributes.product_name.exhaustive": {
+                              "value": "{{query}}"
+                            }
+                          }
+                        }
+                      }
+                    },
+                    {
+                        "bool" : {
+                            "should" : {{#toJson}}rules{{/toJson}}{{^rules}}[ { "match":{"spin":{ "query":"xxxxx"} } }]{{/rules}}
```

Comment:
@raghu-vj  need to update this to a better default. Have added this for now, i think should work functionally correct.